cli/ops/dispatch_json.rs

```diff
@@ -14,6 +14,9 @@ pub type AsyncJsonOp =
 pub enum JsonOp {
   Sync(Value),
   Async(AsyncJsonOp),
+  // AsyncOptional is the variation of Async, which
+  // doesn't block the program exiting.
+  AsyncOptional(AsyncJsonOp),
```

Comment:
"Optional" isn't very descriptive. Maybe we could call it something like `AsyncNoBlock` ?